generator/src/main/java/com/scottlogic/deg/generator/Guice/DecisionTreeWalkerProvider.java

```diff
@@ -28,14 +28,16 @@ public DecisionTreeWalkerProvider(
             public DecisionTreeWalker get() {
           switch(this.configSource.getWalkerType()) {
               case CARTESIAN_PRODUCT:
-                return this.cartesianProductDecisionTreeWalker;
+                  return this.cartesianProductDecisionTreeWalker;
```

Comment:
Minor point, i don't think this file was meant to change - there is only white-space changes apparent.

Reply:
There was a default case added to replace the return null that was there before, is this ok?

Reply:
Also I think the whitespace changes are so that it is now in the correct tabbing format?